apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -386,10 +388,36 @@ Contacts.prototype = {
     }
   },
 
+  checkContactCreated: function() {
```

Comment:
I don't think this name reflects what the function does. I'd rename it to `get contactFirstName()`. What do you think?